assignment4.txt

```diff
@@ -0,0 +1,15 @@
+Mariposa Lee 
+1. In order to put strings together, C uses function strcat(_,_). The first variable would be representing the first string, while the second variable represents the second string. For example, if the first string was "ice" and second was "cream", concatenating the two strings will result in "ice cream". 
```

Comment:
Close - there wouldn't be a space unless ice included a space at the end or cream included a space in the beginning.